### PR TITLE
contrib: added journal custom field.

### PR DESCRIPTION
**Pull request by alejandromumo**
closes https://github.com/inveniosoftware/invenio-rdm-records/issues/1191

- Added `Journal` custom field to zenodo RDM

### CODE REVIEW

File: invenio.cfg [hunk 1]
```diff
@@ -341,3 +341,26 @@ RDM_CUSTOM_FIELDS = [
...
+]
+
+RDM_CUSTOM_FIELDS = [
+    *RDM_CUSTOM_FIELDS,
+    *JOURNAL_CUSTOM_FIELDS,
+]
+
+RDM_NAMESPACES = {
+    **RDM_NAMESPACES,
+    **JOURNAL_NAMESPACE,
```
Comment (zzacharo) [inline invenio.cfg:365]: why don't you add it directly when we define the `RDM_NAMESPACES`?

File: invenio.cfg [hunk 2]
```diff
@@ -341,3 +341,26 @@ RDM_CUSTOM_FIELDS = [
...
+    JOURNAL_CUSTOM_FIELDS_UI
+)
+
+RDM_CUSTOM_FIELDS_UI = [
+    JOURNAL_CUSTOM_FIELDS_UI
+]
+
+RDM_CUSTOM_FIELDS = [
+    *RDM_CUSTOM_FIELDS,
+    *JOURNAL_CUSTOM_FIELDS,
```
Comment (zzacharo) [inline invenio.cfg:360]: Same as below

File: invenio.cfg [hunk 3]
```diff
@@ -341,3 +341,26 @@ RDM_CUSTOM_FIELDS = [
...
+# Journal CF
+
+from invenio_rdm_records.contrib.journal.custom_fields import (
+    JOURNAL_NAMESPACE,
+    JOURNAL_CUSTOM_FIELDS,
+    JOURNAL_CUSTOM_FIELDS_UI
+)
+
+RDM_CUSTOM_FIELDS_UI = [
+    JOURNAL_CUSTOM_FIELDS_UI
```
Comment (zzacharo) [inline invenio.cfg:355]: is this an object or a list?

File: invenio.cfg [hunk 4]
```diff
@@ -341,3 +341,26 @@ RDM_CUSTOM_FIELDS = [
 # Search query of recent uploads
 # Defaults to newest records search
 ZENODO_FRONTPAGE_RECENT_UPLOADS_QUERY = "/api/records?sort=newest&size=10"
+
+
+# Journal CF
+
+from invenio_rdm_records.contrib.journal.custom_fields import (
```
Comment (zzacharo) [inline invenio.cfg:348]: move the imports in the beginning of the file?

**Review by zzacharo — COMMENTED** [review]
@slint probably out of scope but the custom fields are getting out of hand I think in invenio.cfg....maybe we should move them to `site` folder.....

File: invenio.cfg (hunk 3 above)
Comment (alejandromumo) [reply invenio.cfg:355]: `JOURNAL_CUSTOM_FIELDS_UI` is an object, while `RDM_CUSTOM_FIELDS_UI` is a [list](https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/config.py#L405)

File: invenio.cfg (hunk 1 above)
Comment (alejandromumo) [reply invenio.cfg:365]: Nice catch, it was a leftover from testing codemeta + journal at the same time.
I will move it!

File: invenio.cfg (hunk 2 above)
Comment (alejandromumo) [reply invenio.cfg:360]: Nice catch, it was a leftover from testing codemeta + journal at the same time.
I will move it!

**Review by ppanero — APPROVED** [review]
@alejandromumo could you also note down that we might want to move this to site folder? maybe worth asking for a discussion time and also add the naming question.

**Pull request merged**